src/app/core/services/github.service.ts

```diff
@@ -202,4 +203,10 @@ export class GithubService {
   getRepoURL(): string {
     return ORG_NAME.concat('/').concat(REPO);
   }
+
+  viewIssueInBrowser(id: number) {
+    if (id) {
```

Comment:
Sorry I didn't quite catch your logic here. What does `if(id)` mean to indicate?

Reply:
hi, the `if(id)` is to check for null or empty value. Is this part unnecessary since the id is obtained from the table list itself?

Reply:
What would happen if the id is null/undefined?

Reply:
If it is null/undefined the app will not open url in the browser

Reply:
Should we consider showing some kind of message to the user in that case?

Reply:
Okay, I will display an error message for the user

Reply:
I have made the necessary update